File: Python/datastructures/Stack.py
class Stack:
    def __init__(self):
        self.items = []

    def isEmpty(self):
        return self.items == []

    def push(self, item):
        self.items.append(item)

    def pop(self):
        return self.items.pop()

    def size(self):
        return len(self.items)

s = Stack()
s.push(5)
s.push(4)
s.push(3.3)
s.push('A')
s.push('B')
s.push(True)
print(s.size())
print(s.isEmpty())
print(s.pop())
print(s.pop())
print(s.size())

File: Java/Stack/Stack.java
package stack;

public class Stack {
	static final int MAX = 1000;
	public int[] arr = new int[MAX];
	public int top;

	public Stack() {
		top = -1;
	}

	public boolean isEmpty() {
		return top == -1;
	}

	public boolean isFull() {
		return top == MAX - 1;
	}

	public boolean push(int data) {
		if (top >= (MAX - 1)) {
			System.out.println("Stack Overflow");
			return false;
		} else {
			arr[++top] = data;
			System.out.println(data + " pushed into stack");
			return true;
		}
	}

	public int pop() {
		if (top < 0) {
			System.out.println("Stack Underflow");
			return 0;
		} else {
			int x = arr[top--];
			return x;
		}
	}

	public int getSize() {
		return top + 1;
	}
}